src/stepper/Stepper.tsx

```diff
@@ -196,6 +207,7 @@ export default class Stepper extends PureComponent<StepperProps, StepperState> {
           status={ButtonStatus.UNSTYLED}
           disabled={isMin}
           isBubble
+          focus={!isMin && isMax && focus}
```

Comment:
I'm not sure to understand the focus conditions we want here. When do we want to focus the buttons?

Reply:
Removed